core/test/surplus-test.ts

```diff
@@ -77,9 +77,6 @@ describe('Surplus Auction', () => {
         const auctions = await fetchActiveSurplusAuctions('custom');
         const currentAuction = auctions[0];
         expect(currentAuction.id).to.equal(2328);
-        if (currentAuction.state === 'collected') {
-            throw new Error('auction state can not be "collected"');
-        }
```

Comment:
Why exactly this was removed? What introduced changes caused this auction to become `collected`?